assignments/hackyourtemperature/server.js

```diff
@@ -0,0 +1,13 @@
+import express from "express";
+const app = express();
+
+app.get("/", (req, res) => {
+  res.send("hello from backend to frontend!");
+});
+app.use(express.json());
```

Comment:
by habits and in order to make it easier to read, we usually move those kind of statement above the routes definitions such as:

```suggestion
import express from "express";
const app = express();

app.use(express.json());

app.get("/", (req, res) => {
  res.send("hello from backend to frontend!");
});
```